tests/unit/package_managers/test_pip.py

```diff
@@ -2686,23 +2686,41 @@ def test_process_non_existing_package_distributions(
             == f"PyPI query failed: No details about project '{package_name}' available at URL"
         )
 
+    @pytest.mark.parametrize("allow_binary", (True, False))
     @mock.patch.object(pypi_simple.PyPISimple, "get_project_page")
     def test_process_existing_package_without_distributions(
```

Comment:
Is there a test for what happens when there are 0 sdists but there are some wheels?